src/confd/src/infix-services.c

```diff
@@ -370,6 +451,129 @@ static int web_change(sr_session_ctx_t *session, uint32_t sub_id, const char *mo
 	return put(cfg, srv);
 }
 
+/**
+ * Recursive directory remove (rm -r)
+ * @param file or directory to remove
+ *
+ * @returns 0 on success, else error
+ */
+
+int rm(const char *path) {
```

Comment:
This seems like an odd place for such a generic function; is it a candidate for `libite`?

Also, it is quite intricate. Any reason we can't take the approach suggested on [SO](https://stackoverflow.com/a/5467788/715363)?

Reply:
It was an omission from my side, I have a similar function in Finit that I haven't refactored and moved to libite yet.  Maybe we can use `libsrx/src/{helpers,common}.[ch]` for generic helpers that can be migrated?